aws/kms_key/examples/basic/main.tf

```diff
@@ -0,0 +1,6 @@
+module "example" {
```

Comment:
No inputs but module requires inputs - can this be updated to a working example?

Reply:
Ahh good spot! I had overlooked this